src/components/Organization/OrgInfoCard/editOrgDetailsModal.js

```diff
@@ -63,134 +62,175 @@ const EditOrgDetailsModal = ({ currentOrgData, modelCloseCallback }) => {
 
   useEffect(() => {
     if (loading === false && error === false) {
-      message.success("Changes saved!");
       closeModal();
     }
   }, [closeModal, loading, error]);
 
-  useEffect(() => {
-    form.setFieldsValue({
-      org_name: currentOrgData.org_name,
-      org_website: currentOrgData.org_website,
-      org_link_facebook: currentOrgData.org_link_facebook,
-      org_link_github: currentOrgData.org_link_github,
-      org_link_linkedin: currentOrgData.org_link_linkedin,
-      org_link_twitter: currentOrgData.org_link_twitter,
-      org_description: currentOrgData.org_description,
-      org_country: currentOrgData.org_country
-    });
-  }, [form, currentOrgData]);
-
-  const onSubmit = formData => {
+  const [formValue, setformValue] = React.useState({
```

Comment:
Let's directly call useState here since we have imported it in line 1. (Similar to line 26 and 27)

Reply:
Done!